ClassLibrary/clsStaff.cs

```diff
@@ -82,16 +85,34 @@ public string FamilyName
             }
         }
 
-         public bool Find(int ID)
+         public bool Find(int StaffID)
          {
-             StaffID = 11;
-             StaffPassword = "password";
-             IsManager = true;
-             DOB = DateTime.Now;
-             FirstName = "first name";
-             FamilyName = "family name";
+            // Creates a new instance of the data connection.
+            clsDataConnection DB = new clsDataConnection();
```

Comment:
I created a static database reference so that you don't have to set it up every time. It's the top field in the clsDataConnection class, I think it's called connection

Reply:
Will change once I fetch.